enacts/flex_fcst/cpt.py

```diff
@@ -106,6 +106,32 @@ def starts_list(
     return start_dates
 
 
+def read_mpycptv2dataset(data_path):
```

Comment:
Rename this `read_pycptv2dataset` and rename the current `read_pycptdataset` to `read_pycptdataset_one_season`?

Reply:
I used _target rather than _season in case we are able to adapt this to subseasonal, where targets are week(s), days...